packages/kafkajs-receive/tests/kafkajs.test.ts

```diff
@@ -140,7 +140,6 @@ async function setupTopics(kafka: Kafka, topics: string[]) {
   }
 }
 
-// eslint-disable-next-line @typescript-eslint/no-floating-promises
 suite('kafkajs-receive', async () => {
```

Comment:
Note, I realized while I was working on the compiler tests that the node test suite is fully async. It's ortoginal to this PR, but I updated the kafka tests to use await or .catch in order to remove eslint disablments